src/Monitor.vala

```diff
@@ -4,15 +4,35 @@ namespace Monitor {
         private MainWindow window = null;
         public string[] args;
 
-        construct {
-            application_id = "com.github.stsdc.monitor";
+        private static bool start_minimized = false;
+        private static bool status_minimized = false;
+    private const GLib.OptionEntry[] cmd_options = {
```

Comment:
```suggestion
        private const GLib.OptionEntry[] cmd_options = {
```

@raibtoffoletto BTW it seems that I was missing another 4 spaces in my suggestion. Could you fix this again?